code/game/machinery/_machinery.dm

```diff
@@ -930,6 +930,13 @@
 	if(!machine_board)
 		return FALSE
 
+	/**
+	 * sorting is very important especially because we are breaking the loop on line 1006
```

Comment:
Please point this to the specific piece of the proc, the lines are going to change.

Reply:
Done